core/src/main/scala/com/gu/tableversions/core/model.scala

```diff
@@ -100,3 +100,10 @@ final case class TableDefinition(name: TableName, location: URI, partitionSchema
 sealed trait TableVersion
 final case class PartitionedTableVersion(partitionVersions: Map[Partition, Version]) extends TableVersion
 final case class SnapshotTableVersion(version: Version) extends TableVersion
+
+case class FileFormat(name: String) extends AnyVal
```

Comment:
Yeah I guess to Tom's point, is there a reason we don't want to use enumeratum here?

Reply:
As mentioned above, I think this library should be agnostic about what file formats client code uses. If our data lake jobs are restricted to a limited set, then I think that policy should be enforced in ophan-data-lake, not here.

Reply:
Oh and just to add to that, the set of formats supported by Spark is open-ended (you can provide a custom format), so I don't think it makes much sense to try to enumerate all known Spark file formats here either.

Reply:
TIL!